src/search/lp/lp_solver.cc

```diff
@@ -32,11 +32,20 @@
 #include <cassert>
 #include <iostream>
 #include <numeric>
+#include <unordered_set>
 
 using namespace std;
 using utils::ExitCode;
 
 namespace lp {
+#ifndef NDEBUG
```

Comment:
Do you get an "unused function" error without this? If not, I'd reduce the number of preprocessor directives and just always define the function (it shouldn't have a performance impact if it is not used). I have no strong opinion, though.

Reply:
No, I thought that's something we do if something's used only in debug mode. I'll remove it.